### ISSUE TITLE
npm run test failed

### ISSUE THREAD

**Issue by amit-kaushik** [codeforequity-at/testmybot#96]
After setting up the configuration, tried to run test cases.
It showed error : 
ERR! getUserInfo { __ar: 1,
ERR! getUserInfo   error: 1357004,
ERR! getUserInfo   errorSummary: 'Sorry, something went wrong',
ERR! getUserInfo   errorDescription: 'Please try closing and re-opening your browser window.',
ERR! getUserInfo   payload: null,
ERR! getUserInfo   bootloadable: {},
ERR! getUserInfo   ixData: {},
ERR! getUserInfo   bxData: {},
ERR! getUserInfo   gkxData: {},
ERR! getUserInfo   qexData: {},
ERR! getUserInfo   lid: '6608887005208310168' }[](url)

**Comment by amit-kaushik** [comment]
Yes correct. It's in fbdirect. I thought its related to my FB account but it was not.
I can easily log in to messenger and chat with my bot, but when I use same credentials for the testmybot then it shows this error. I inspected the page also, see screenshot if it helps.
![screen shot 2018-10-06 at 10 14 41 am](https://user-images.githubusercontent.com/43472414/46582234-50bcf780-ca61-11e8-8e80-b3268541f016.png)
![screen shot 2018-10-05 at 10 19 27 pm](https://user-images.githubusercontent.com/43472414/46582236-531f5180-ca61-11e8-86e4-b291ed3ad1c0.png)

**Comment by codeforequity-at** [comment]
This is really hard to say. It seem to me that facebook rejects the login.
In general, the "fbdirect" mode is instable, as it basically hacks some facebook internal APIs.
I would recommend to switch to the new webdriver connector (which simulates clicking through with a real browser).

**Comment by amit-kaushik** [comment]
I resolved the same issue by using another account for testing purposes. Is it something related to particular account that I used ? and what is new Webdriver Connector ?

**Comment by codeforequity-at** [comment]
I had some strange problems with this as well ... sometimes my account became locked, depending on the workstation I used ... that's why I cannot recommend to do it this way.
the webdriver connector uses Selenium under the hood to test a chatbot directly on messenger.com
